src/rhub/openapi/bare_metal/image.yml

```diff
@@ -0,0 +1,182 @@
+model:
+  BareMetalImage:
+    type: object
+    properties:
+      id:
+        allOf:
+          - $ref: '../common.yml#/model/ReadOnlyID'
+      version:
+        allOf:
+          - $ref: '../common.yml#/model/String128'
+      description:
+        type: string
+      base_os:
+        allOf:
+          - $ref: 'common.yml#/components/ImageBaseOS'
+      arch:
+        allOf:
+          - $ref: 'common.yml#/components/BareMetalArch'
+      legacy_bios:
+        allOf:
+          - $ref: '../common.yml#/model/BooleanDefaultTrue'
+      uefi:
+        allOf:
+          - $ref: '../common.yml#/model/BooleanDefaultTrue'
+      secure_boot:
```

Comment:
Do you think we should be consistent and previx with uefi_secure_boot?  Up to you.